### PR TITLE
changed database from userdata to user

### PR DESCRIPTION
**Pull request by divyansh12git**
Changes Made

This pull request introduce changing  of "UserData" collection to "User" collection from firebase. 

1).The data is fetched for the current user using FirebaseAuth.instance.currentUser, 
2).inside that user we have Account collection, which is used in stream and all the docs had been fetched.
3). The Widget MyListViewBuilder in homepage.dart is correspondingly changed.

db
AD

### CODE REVIEW

File: lib/books model/issuedBookHomePage.dart [hunk 1]
```diff
@@ -5,7 +5,7 @@ class IssuedBookModel{
   final String ReturnDate;
   final String authorName;
```
Comment (piyuuussh) [inline lib/books model/issuedBookHomePage.dart:6]: This model is not being used so no need of this in the code.

File: lib/home_screen.dart [hunk 2]
```diff
@@ -277,18 +293,11 @@ class _HomePageState extends State<HomePage> {
...
-                                  size: 42,
-                                  color: Color.fromARGB(178, 32, 39, 78),
-                                ),
-                              )),
-                          Text('Add'),
-                          Text('Books')
+                                  onTap: () {},
+                                  child: Image.asset(
+                                      'assets/images/nibuslib.png'))),
+                          Text('K-Nimbus'),
```
Comment (piyuuussh) [inline lib/home_screen.dart:299]: Why is add book section being removed the Code .

File: lib/home_screen.dart [hunk 3]
```diff
@@ -149,7 +165,7 @@ class _HomePageState extends State<HomePage> {
                 padding: EdgeInsets.only(top: 130, left: 38),
                 child: Container(
                   child: Text(
-                    documentData["Username"],
```
Comment (piyuuussh) [inline lib/home_screen.dart:152]: why is divyansh gupta is being used as universal name instead it should be showing the name of the user whose account is logged in.

**Review by piyuuussh — CHANGES_REQUESTED** [review]
Make all the necessary changes
image

**Review by ShubhamTiwari55 — COMMENTED** [review]
Make all changes as requested by piyush...

**Pull request closed without merging**